app/main.py

```diff
@@ -1,13 +1,24 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+
+    return {
+        "errors" : [format_linter_error(error) for error in errors],
+        "path" : file_path,
+        "status" : "".join(["failed" if errors else "passed"][0])
```

Comment:
u don't need to use join here
```suggestion
        "status" : "failed" if errors else "passed"
```

Reply:
I fixed this